app/frontend/src/cljs/teet/contract/contract_common.cljs

```diff
@@ -12,18 +12,23 @@
             [teet.contract.contract-status :as contract-status]
             [teet.ui.format :as format]
             [teet.util.euro :as euro]
-            [teet.user.user-model :as user-model]))
+            [teet.user.user-model :as user-model]
+            [reagent.core :as r]))
 
 (defn contract-procurement-link
   [{:thk.contract/keys [procurement-number]}]
-  (if (js/Number.isInteger (js/parseInt (subs procurement-number 0 1)))
-    [common/external-contract-link
-     {:href (str (environment/config-value :contract :state-procurement-url)
-              procurement-number "/general-info")}
-     (str (tr [:contracts :state-procurement-link]) " " procurement-number)]
-    [:span (str (tr [:contracts :state-procurement-link]) " " procurement-number)]))
+  (r/with-let [procurement-display-text (str (tr [:contracts :state-procurement-link]) " " procurement-number)]
+    (if (js/Number.isInteger (js/parseInt (subs procurement-number 0 1)))
+      [common/external-contract-link
+       {:href (str (environment/config-value :contract :state-procurement-url) procurement-number "/general-info")}
+       procurement-display-text]
+      [typography/SmallText {:style {:display :flex
+                                     :align-items :center
+                                     :margin-right "1rem"}}
+       procurement-display-text])))
 
 (defn contract-external-link
+  [{:thk.contract/keys [external-link procurement-number]}]
```

Comment:
this seems like a typo? or accidental copy of the parameter row.

Reply:
totally missed that. Fixed now